docs/porting/third-party-deps.md

```diff
@@ -1,19 +1,69 @@
 # Analyzing your 3rd Party Dependencies
 
-TODO intro
+The first step in the porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
 
 ## Prerequisites
 
-TODO prerequisites
+This article will assume you are using Windows and Visual Studio.  For your NuGet package dependencies, you may also wish to download the [Nuget Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer).
```

Comment:
You might motivate why you need NPE. 

Try "It's much easier if" or "The easiest way to explore the contents and framework support of NuGet packages is by using ..."